cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1007,6 +1007,9 @@ private boolean handleSSLFailure(Certificate[] lastChain) throws CommandLineExce
                 response = readLine("Accept certificate? [N]o, [T]emporarily : ", false, true);
             }
 
+            if (response == null)
+                return false;
+
             if (response != null && response.length() == 1) {
```

Comment:
This can become

'''
if (response == null) {
    return false;
} else if (response.length() == 1) {
```